File: code_samples/python/v1.0.0/update_payment.py
from tatrapayplus import TBPlusSDK
from tatrapayplus.models import *

client = TBPlusSDK(
    "your-client-id",
    "your-client-secret"
)
payment_id = "b54afd37-5bb9-4080-9416-5ec450779087"
update_data = CardPayUpdateInstruction(
    operation_type=CardPayUpdateInstructionOperationType.CHARGEBACK,
    amount=120,
)
update_response = client.update_payment(payment_id, update_data)

File: code_samples/node/v1/updatePayment.js
const paymentId = "b54afd37-5bb9-4080-9416-5ec450779087";
const cancelPreAuthorization = await sdk.updatePayment(paymentId, {
  operationType: "CANCEL_PRE_AUTHORIZATION",
});
// or
const confirmPreAuthorization = await sdk.updatePayment(paymentId, {
  operationType: "CONFIRM_PRE_AUTHORIZATION",
});
// or
const chargeBackResponse = await sdk.updatePayment(paymentId, {
  operationType: "CHARGEBACK",
  amount: 100,
});
